app/scripts/views/sign_in_totp_code.js

```diff
@@ -0,0 +1,86 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
```

Comment:
in this new file try to drop 

```
+define(function (require, exports, module) {
+  'use strict';
```

and just have `module.exports = View;` , it should just work now (with webpack)